apps/alpha-test-app/src/components/layout-demo/PageDemo.vue

```diff
@@ -0,0 +1,69 @@
+<script lang="ts" setup>
+import { OnyxHeadline } from "sit-onyx";
+import { FloatingButtonDemo, FlyoutDemo, LayoutSettings, StickyDemo, TooltipDemo } from ".";
+import type { SettingsSections } from ".";
+import { computed } from "vue";
+
+const settings = defineModel<SettingsSections>();
+
+const props = defineProps<{
+  isSidebarOpen: boolean;
+}>();
+const emit = defineEmits<{
+  isSidebarOpen: [value: boolean];
+}>();
+const isSidebarOpen = computed({
+  get: () => props.isSidebarOpen,
+  set: (value) => emit("isSidebarOpen", value),
+});
+
+const muchContent = Array.from({ length: 100 }, (_, index) => `Lorem ipsum dolor ${index}`);
+
+const showSidebarOpenButton = computed<boolean>(() => {
+  if (!settings.value) return false;
+  const { showSidebarCollapse, showTempOverlayTransparent, showTempOverlay } =
+    settings.value.sidebar;
+  return (
+    (!props.isSidebarOpen &&
+      (showSidebarCollapse || showTempOverlayTransparent || showTempOverlay)) ||
```

Comment:
```suggestion
      (showSidebarCollapse || showTempOverlayTransparent || showTempOverlay)) ??
```

I think `??` is safer here because `||` with booleans can cause unwanted effects